src/modules/search-time/selector/index.tsx

```diff
@@ -16,6 +16,8 @@ type SearchTimeSelectorProps = {
   options?: SearchMode[];
 };
 
+const yesterday = format(subDays(new Date(), 1), 'yyyy-MM-dd');
+
```

Comment:
I think this variable should be declared inside the `SearchTimeSelector`. Let me know if there is a reason why it should not and the reason for being created here 👍